test/components/about/aboutpage.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { mount } from "enzyme";
+import AboutPage from "../../../pages/about";
+//data
+import info from "./../../../public/data/info.json";
+import links from "./../../../public/data/links.json";
+import members from "./../../../public/data/members.json";
+
+describe("About page", () => {
+  const wrapper = mount(<AboutPage />);
+  it("contains correct infos", () => {
+    for (const i of info) {
```

Comment:
If we rename `info` to something like `about_page_infos`, then we should consider renaming this variable from `i` to something more descriptive.